src/mixins/themeable.ts

```diff
@@ -11,15 +11,23 @@ import { assign } from '@dojo/core/lang';
  * to be applied where each class in appliedClasses
  * is used.
  */
-export type CSSModuleClassNames = {
+export type FlaggedCSSModuleClassNames = {
```

Comment:
Not obvious when reading this what Flagged is or means... flagged by what or in what way?